doc/design/vic-machine.md

```diff
@@ -63,6 +63,111 @@ To create a VCH, disambiguation of which resource to use from the available set
   - volumes
 
 
+## Usage Examples
+
+These are various examples intended to illustrate how vic-machine is intended to be used. These are illustrations rather than the finalized naming so if something doesn't make sense ask about it. 
+
+### Fully specified command line
+
+```
+vic-machine create
+-TO=https://root@****:vcenter/
+-name=edw-web
+-compute-resource=/example-datacenter/host/mycluster/Resources/edw-pool/web
+-image-store=ds://datastore/containers/edw
```

Comment:
What does this format mean? Is the containers/edw the directory in datastore? How does this work in vSAN?